public/app/plugins/panel/heatmap/module.tsx

```diff
@@ -409,4 +409,5 @@ export const plugin = new PanelPlugin<Options, GraphFieldConfig>(HeatmapPanel)
       category,
     });
   })
-  .setSuggestionsSupplier(new HeatmapSuggestionsSupplier());
+  .setSuggestionsSupplier(new HeatmapSuggestionsSupplier())
+  .setDataSupport({ annotations: true });
```

Comment:
i wonder how this will work when when x != time :thinking: , i guess we'll just have to omit the annotation-related stuff in the panel when the data has no time field...or something.